onosproject/onos-docs
# Automate building docs locally and remotely and also improve its workflow

To address issue #23 and allow the users to build docs locally as well for testing purposes. Travis also has a default provider that publishes static websites automatically using travis CI bot so we don't need to create a bot github user. 

To test it locally:
run `make docs-serve` and then check this address from your browser: `localhost:8000` 

After this PR, I will take care of versioning. To find out if it works properly, we need to merge (local test works fine). 

P.S. For now, I disabled html and markdown format checking but later we should enable it.
